docs/user-guide/creating-application/workflow/ci-pipeline.md

```diff
@@ -28,13 +48,18 @@ In this column, you have to provide your branch or tag regex where your code is
 
 **Example**- It can be a master branch in the production and development/staging branch in the dev environment. But the branch name completely depends on the user, which branch name the user wants to provide here.
 
-### 4. Stages
+### 4. Stages: 
+There are 3 dropdowns given below:
+
+* **Pre-build**
+* **Docker build**
+* **Post-build**
 
 **\(a\) Pre-build**
 
-This section is used for those Scripts which you want to execute before building the Docker image. To add a Pre-build stage, click on Add Stage and provide a name to your pre-stage and write your script as per your requirement. This will run in sequence before the docker image is built. You can also provide the path of the directory where the output of the script will be stored in the Remote Directory column. But this is optional to fill because many times you run scripts which do not provide any output.
+This section is used for those Scripts which you want to execute before building the Docker image. To add a Pre-build stage, click on `Add Stage` and provide a name to your pre-stage and write your script as per your requirement. This will run in sequence before the docker image is built. You can also provide the path of the directory where the output of the script will be stored in the `Remote Directory` column. But this is optional to fill because many times you run scripts which do not provide any output.
```

Comment:
This section is used for those steps which you want to execute before building the Docker image. To add a Pre-build stage, click on `Add Stage` and provide a name to your pre-stage and write your script as per your requirement. These stages will run in sequence before the docker image is built. Optionally, you can also provide the path of the directory where the output of the script will be stored locally.